### DISCUSSION TITLE
Migrate database of Photoprism or install using MariaDB database

### DISCUSSION THREAD

**Discussion by besforttt** [tteck/Proxmox#1715] [Help Needed]
All the online documentation I can find, it only references docker install method!
Being able to switch to MariaDB from SQLite3 would be awesome for me, it's the third time I keep re-indexing images (around 70k) only to have something go wrong with sqlite3.
The resources are not an issue as I've allocated 16 vCores, 512GB NVMe SSD and 32GB of RAM to it!

If a migration is not possible, I am willing to re-install from scratch just as long as I'd be able to use MariaDB as a database server.

Thanks,
besforttt

**Comment by shbatm** [comment]
Can you follow the same instructions as docker and just use the LXC shell as your 'host' and start/stop with `service photoprism start/stop` (it's installed as a systemd service).

Getting ready to jump in to this and came across your question. That's what I'm going to be attempting with installing mariadb inside the LXC from apt. 

Config variables for PhotoPrism look to be in `/var/lib/photoprism/.env`
